components/test_storage/src/assert_storage.rs

```diff
@@ -233,7 +233,10 @@ impl<E: Engine> AssertionStorage<E> {
         }
     }
 
-    fn expect_invalid_tso_err(&self, resp: Result<(), storage::Error>, sts: u64, cmt_ts: u64) {
+    fn expect_invalid_tso_err<T>(&self, resp: Result<T, storage::Error>, sts: u64, cmt_ts: u64)
+    where
+        T: std::fmt::Debug,
+    {
```

Comment:
Why is this change needed?

Reply:
This function previously accepts Result<()>, and is used on [commit](https://github.com/tikv/tikv/pull/5966/files#diff-e75ef022cc94f1afdbe5fc6a53843b4fR453) and some other function's return values. Now the commit's result has changed from () to TxnStatus, so to make this function still work I made it generic.